svgelements/svgelements.py

```diff
@@ -6021,21 +6021,44 @@ def reverse(self):
         self._segments[0].start = prepoint
         return self
 
+    def _subpath_indices(self):
+        """
+        Returns the indexes of Move segments assuming that the first segment is a Move.
+
+        This can be used to count subpaths or to get segment start and end numbers for subpaths.
+        """
+        if len(self._segments) == 0:
+            raise StopIteration
```

Comment:
I think Python 3.9 or 3.10 or even 3.8 start throwing a fit with you raising a StopIteration manually here. The correct method is to return at this point.

Reply:
Agreed.